lib/watch-files.js

```diff
@@ -1,17 +1,40 @@
 const chokidar = require('chokidar');
-const { isEmpty } = require('omnibelt');
+const { isEmpty, forEachSerialP } = require('omnibelt');
 const { logError, logResult, loadConfig, unlockConfig } = require('./utils');
 const paramsByCommand = require('./get-upload-params');
 const getUploader = require('./get-uploader');
 const path = require('path');
 
+const queueInfo = {
+  queue: [],
+  queueData: new Map(),
+  isRunning: false
+};
+
+setInterval(async () => {
+  const { queue, queueData } = queueInfo;
+  let { isRunning } = queueData;
+  if (queue.length && !isRunning) {
+    isRunning = true;
+    await forEachSerialP(async () => {
+      const file = queue.pop();
```

Comment:
use shift instead of pop to do FIFO. it feels weird to have things saved backwards for some reason. 
```
Started watching ::	experience
change	occurred on experience/components/gaTracking.hbs, queueing upload
change	occurred on experience/components/errorAlert.hbs, queueing upload
upload	processing queue
processing	experience/components/errorAlert.hbs
uploaded	experience/components/errorAlert.hbs
upload	processing queue
processing	experience/components/gaTracking.hbs
uploaded	experience/components/gaTracking.hbs
```